.emacs.d/test/lisp/run-tests.el

```diff
@@ -4,4 +4,17 @@
 
 (load-file (concat (file-name-directory load-file-name) "/dot-tests.el"))
 
+(when (require 'undercover nil 'noerror)
+  ;; undercover can't search for matching files recursively
+  ;; override a local function that fixes this
+  (advice-add
+   #'undercover--wildcards-to-files :override
+   (lambda (wildcards)
```

Comment:
I'm overriding a function that takes an argument so I need to allow an argument to be passed.